frontend design suggestions

rlangenf opened the pull request:
(WIP, more to come)

Commit:
erstes frontend zeug (hintergrund kommt noch)

Barathaner commented on client/src/App.css:
```diff
@@ -7,9 +7,10 @@
   flex-direction: column;
   font-family: "Nunito", sans-serif;
-  background-image: linear-gradient(90deg, #410076, #8820dd);
+  background-image: linear-gradient(#F0F0F0, #F0F0F0, #8b8b8b);
```
Lila schön weiß aua auge

rlangenf replied:
Hab ich gemacht

Barathaner commented on client/src/App.js:
```diff
@@ -12,7 +12,7 @@ function App() {
 
     function initAllMods() {
-        fetch("/getall").then(
+        fetch("https://unikarlender.de/getall").then(
```
für PR wieder zurück

Barathaner commented on client/src/App.js:
```diff
@@ -23,7 +23,7 @@ function App() {
     function postdata(nameslol) {
 
-        fetch('/moduleswithevents', {
+        fetch('https://unikarlender.de/moduleswithevents', {
```
für PR wieder zurück

Commit:
more frontend suggestions

- nested application grid inside a container (see https://getbootstrap.com/docs/5.0/layout/grid/)
- adjusted button and footer positioning
- added Bootstrap Collapse Component to accordion
- renamed CSS-classes in AccordionItem
- some formatting in CSS-Files

Barathaner commented:
Awesome work

Barathaner commented:
![image](https://user-images.githubusercontent.com/40422666/203608919-86eab087-a838-42fe-9f2a-cfd03beff03e.png)

can you make this animated rotation  when it opens up?
